index.js

```diff
@@ -107,9 +110,15 @@ async function getRepoFromArguments() {
 
   console.log('Pull requests deployed to staging since the last deployment to production are:\n');
 
+  const uniqueUsers = new Set()
+
+  let count = 0;
   for (const pr of newPRs) {
-    console.log(`- ${pr.user.login}: ${pr.title}`);
+    uniqueUsers.add(pr.user.login)
+    //console.log(`(${++count}) \x1B[36m${pr.user.login}: \x1B[0m${pr.title} \x1B[32m(${DateTime.fromISO(pr.merged_at).toLocaleString(DateTime.DATETIME_MED)})\x1B[0m`);
+    console.log(`(${++count}) ${chalk.blue(pr.user.login + ':')} ${pr.title} ${chalk.gereen(DateTime.fromISO(pr.merged_at).toLocaleString(DateTime.DATETIME_MED))}`)
```

Comment:
Using a template literal here actually reduces readability. I believe there's a more readable way of composing this with `chalk`.